@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -198,6 +229,11 @@ void main() {
     // Compute distance to dash edge (0.5: dashCenter, 0.0: dashEdge) and compute the
     // dashBlendFactor similarly on how we did it for the line opacity.
     float distToDashEdge = 0.5 - distance(distToDashOrigin, (d + g) / d * 0.5);
+    #if DASHES_MODE == DASHES_ROUND
```

Comment:
No handling for DASHES_MODE == DASHES_NONE. How does it differ from DASHES_SQUARE?

Reply:
Removing DASHES_NONE, as it is not used anywhere. Historically the dashes are activated (#define USE_DASHED_LINE) by setting "gapSize" > 0.

Reply:
Ok got it!